<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportTreatmentsSetupDialog</class>
 <widget class="QDialog" name="ReportTreatmentsSetupDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>412</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item row="1" column="1" colspan="3">
    <widget class="CDateEdit" name="edtEndDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblPurpose">
     <property name="text">
      <string>Назначение</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbPurpose"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblMedicalAidType">
     <property name="text">
      <string>Тип помощи</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1" colspan="4">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbMedicalAidType"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата &amp;начала периода</string>
     </property>
     <property name="buddy">
      <cstring>edtBegDate</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата &amp;окончания периода</string>
     </property>
     <property name="buddy">
      <cstring>edtEndDate</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="CDateEdit" name="edtBegDate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="displayFormat">
      <string>dd.MM.yyyy</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblEventType">
     <property name="text">
      <string>&amp;Тип обращения</string>
     </property>
     <property name="buddy">
      <cstring>cmbEventType</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbEventType"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblFinance">
     <property name="text">
      <string>Тип финансирования</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="4">
    <widget class="CRBComboBox" name="cmbFinance"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="text">
      <string>Подразделение</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="btnPersonList">
     <property name="text">
      <string>Врач</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="4">
    <widget class="QLabel" name="lblPersonList">
     <property name="text">
      <string> не задано</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lblTariffing">
     <property name="text">
      <string>Тарификация</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1" colspan="4">
    <widget class="QComboBox" name="cmbTariffing">
     <item>
      <property name="text">
       <string>Не учитывать</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Учитывать</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lblAge">
     <property name="text">
      <string>Возраст с</string>
     </property>
     <property name="buddy">
      <cstring>edtAgeFrom</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QSpinBox" name="edtAgeFrom">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QLabel" name="lblAgeTo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>по</string>
     </property>
     <property name="buddy">
      <cstring>edtAgeTo</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="3">
    <widget class="QSpinBox" name="edtAgeTo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximum">
      <number>150</number>
     </property>
    </widget>
   </item>
   <item row="11" column="4">
    <widget class="QLabel" name="lblAgeYears">
     <property name="text">
      <string>лет</string>
     </property>
     <property name="buddy">
      <cstring>edtAgeTo</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="13" column="0" colspan="5">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btnEventTypeList">
     <property name="text">
      <string>Тип обращения</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="QLabel" name="lblEventTypeList">
     <property name="text">
      <string>Не задано</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="btnSpecialityList">
     <property name="text">
      <string>Специальность</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="4">
    <widget class="QLabel" name="lblSpecialityList">
     <property name="text">
      <string>не задано</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>cmbPurpose</tabstop>
  <tabstop>cmbMedicalAidType</tabstop>
  <tabstop>btnEventTypeList</tabstop>
  <tabstop>cmbEventType</tabstop>
  <tabstop>cmbFinance</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>btnPersonList</tabstop>
  <tabstop>cmbTariffing</tabstop>
  <tabstop>edtAgeFrom</tabstop>
  <tabstop>edtAgeTo</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ReportTreatmentsSetupDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ReportTreatmentsSetupDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
